src/t_stream.c

```diff
@@ -67,13 +71,26 @@ stream *streamNew(void) {
     return s;
 }
 
+static void streamLpFreeGeneric(void *lp, void *str) {
+    stream *s = str;
```

Comment:
str seems confusing.

```suggestion
static void streamLpFreeGeneric(void *lp, void *stream) {
    stream *s = stream;
```

Reply:
Naming the parameter `stream` shadows the type and doesn't compile.
Renamed `str` -> `stream_` in [60c4954](https://github.com/redis/redis/pull/14363/commits/60c495409cefb6746f368be653ecf1b6b17c8c50)

Reply:
by custom we don't use the name as this way(_ at the end).